test/sql/pragma/test_pragma_output.test

```diff
@@ -0,0 +1,18 @@
+# name: test/sql/pragma/test_pragma_output.test
+# description: Test last_profiling_output, detailed_profiling_output and all_profiling_output pragma
+# group: [pragma]
+
+statement ok
+CREATE TABLE integers(i INTEGER DEFAULT 1+3, j INTEGER)
+
+statement ok
+SELECT * FROM pragma_last_profiling_output()
+
+statement ok
+SELECT * FROM pragma_detailed_profiling_output()
+
+statement ok
+PRAGMA all_profiling_output
+
+statement ok
+PRAGMA set_profiler_history_size = 5
```

Comment:
The history size is set but not enough queries are run to actually trigger the history limit from being relevant. Could you run more queries here, mixed with calls to fetch the profiling output? That way the behavior would actually get tested.